docs/source/contributing.rst

```diff
@@ -50,9 +57,37 @@ And then
 
 .. code-block:: none
 
-   python -m pip install -e .[dev]
+   python -m pip install -e ".[dev]"
+
+If you have any trouble with the build step, please open an issue in the
+`dask-ml issue tracker <https://github.com/dask/dask-ml/issues>`_.
+
+Running tests
+~~~~~~~~~~~~~
+
+Dask-ml uses `py.test <https://docs.pytest.org/en/latest/>`_ for testing. You
+can run tests from the main dask-ml directory as follows:
+
+.. code-block:: none
+
+    py.test tests
+
+Alternatively you may choose to run only a subset of the full test suite. For
+example to test only the preprocessing submodule we would run tests as follows:
+
+.. code-block:: none
+
+    py.test tests/preprocessing
+
+
+In addition to running tests, dask-ml verifies code style uniformity with the
+``flake8`` tool:
+
+.. code-block:: none
+
+    pip install flake8
+    flake8 dask_ml
```

Comment:
Could you change this to just `flake8`, so that the tests are linted too.

Side note: do either of you, or anyone else, have thoughts on using https://github.com/ambv/black to just auto-format everything for us? It'd be nice to not have to worry about style issues.

Reply:
> Could you change this to just flake8, so that the tests are linted too.

Yeah, for sure. I'll update the circle builds too. 

> do either of you, or anyone else, have thoughts on using https://github.com/ambv/black to just auto-format everything for us?

I don't have any experience using black. Having code formatted automatically does seem appealing though.